Automattic/wp-calypso
# Update/woop cta

Todo:
- Confirm step in the new installer needs to handle existing atomic sites better / should be handling the install step

#### Changes proposed in this Pull Request

* Points landing page to `/start/woocommerce-install?site=example.com` when not already atomic
* Fixes redirect_to param
* Continues to handle software install in the legacy landing page code
* Deletes the controller
* ^ Removes redirect to woocommerce-admin - There isn't really need for blocking access to this even if you have woo installed, it should just retrigger the woop install process, could be useful to "repair" sites one off if we add new plugins to the set later
* ^ Removes redirect for non business/atomic
* ^ Adds redirect for p2 sites or sites without a plan (e.g. domain only sites, jetpack sites)

#### Testing instructions

Check these:
* Atomic site on business plan => installs woo using legacy installation
* Simple site on business plan => sends you to new installer
* Simple site on premium => sends you to checkout and back to landing page where new installer will now load if you click again

Check these redirect: 
* Jetpack site
* p2 site
* domain only site

Check if we need anything else after anyone responds to: p1638330066305700-slack-C02DQP0FP

Related to #
